dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -69,6 +69,7 @@ RUN pip install cffi==1.10.0 \
                 tabulate \
                 textfsm \
                 virtualenv \
+		retry \
```

Comment:
please make the ident aligned.

Reply:
Please use white spaces instead of tabs. If this was the problem.

Reply:
@keboliu @yxieca I fixed the indentation in file